addons/web/static/src/views/fields/parsers.js

```diff
@@ -25,6 +25,38 @@ function evaluateMathematicalExpression(expr, context = {}) {
     return evaluateExpr(safeEvalString, context);
 }
 
+function parseOperation(value, parseValueFn) {
+    const match = value.match(/^(?<operator>[%+\-*/])=(?<operand>-?\d+(?:[.,]\d+)?)$/);
```

Comment:
remove % as no longer supported

Reply:
decimal and thousands separators are necessarily `.` and `,`, see localization.decimalPoint and localization.thousandsSep

Reply:
I think the regex should allow expressions like `+= 10` (with whitespace)